kolibri/content/test/test_content_app.py

```diff
@@ -334,6 +337,35 @@ def test_channelmetadata_content_available_field_false(self):
         response = self.client.get(reverse("channel-list"))
         self.assertEqual(response.data[0]["available"], False)
 
+    def test_channelmetadata_file_sizes_filter_has_total_resources(self):
+        response = self.client.get(reverse("channel-list"), {"file_sizes": True})
+        self.assertEqual(response.data[0]["total_resources"], 1)
+
+    def test_channelmetadata_file_sizes_filter_has_total_file_size(self):
+        content.LocalFile.objects.filter(files__contentnode__channel_id=self.the_channel_id).filter(renderable_local_files_q_filter).update(file_size=1)
+        response = self.client.get(reverse("channel-list"), {"file_sizes": True})
+        self.assertEqual(response.data[0]["total_file_size"], 2)
+
+    def test_channelmetadata_file_sizes_filter_has_on_device_resources(self):
+        response = self.client.get(reverse("channel-list"), {"file_sizes": True})
+        self.assertEqual(response.data[0]["on_device_resources"], 1)
+
+    def test_channelmetadata_file_sizes_filter_has_on_device_file_size(self):
+        content.LocalFile.objects.filter(files__contentnode__channel_id=self.the_channel_id).filter(renderable_local_files_q_filter).update(file_size=1)
```

Comment:
Hrm, should probably set this on all localfiles, otherwise the test is not telling much.